### PR TITLE
https://github.com/stolostron/backlog/issues/#26974--cluster lifecycl…

### PR DESCRIPTION
**Pull request by swopebe**
…e known issues become MCE cluster lifecycle known issues

### CODE REVIEW

File: release_notes/known_issues.adoc [hunk 1]
```diff
@@ -275,7 +275,13 @@ The cluster claim is re-created automatically.
 [#cluster-management-issues]
 == Cluster management known issues
 
-See the following known issues and limitations for cluster management:
+Cluster management or _Cluster lifecycle_ is provided by the {mce-short} with or without {product-title-short}. See the following known issues and limitations for Cluster management that apply to {product-title-short} only. Most cluster management known issues are located in the Cluster lifecycle documentation at link:../clusters/release_notes/known_issues.adoc#known-issues-cluster[Cluster lifecycle known issues].
```
Comment (swopebe) [inline release_notes/known_issues.adoc:278]: Need peer review on this added paragraph 278.

Comment (oafischer) [inline release_notes/known_issues.adoc:278]: Do we need `the` before `mce-short` here?

**Review by oafischer — APPROVED** [review]
One comment, otherwise looks good.

File: release_notes/known_issues.adoc (hunk 1 above)
Comment (swopebe) [reply release_notes/known_issues.adoc:278]: Yes

**Pull request merged**